templates/ChoiceSchema.stencil

```diff
@@ -0,0 +1,6 @@
+
+public enum {{ choice.language.default.name }}: {{ choice.choiceType.language.default.name }}, Codable {
```

Comment:
This should just be `choice.name` and `choice.choiceType.name`. This ensures that it tries to pull from `language.swift` first and then `language.default` only as a backup.